celery/app/defaults.py

```diff
@@ -261,6 +261,7 @@ def __repr__(self):
         inherit_parent_priority=Option(False, type='bool'),
         default_delivery_mode=Option(2, type='string'),
         default_queue=Option('celery'),
+        default_queue_type=Option('classic', type='string'),
```

Comment:
@Nusnus Hey, do you remember why you defaulted to `classic`?

Reply:
> @auvipy I'm missing your point. In past releases, yes. It became the recommended queue type in 2024.

it is for legacy celery projects in the wild and enterprise environment. you don't need to worry about it.

Reply:
I am not sure why you are trying to shut me down but we can do better than this. You don't need respond if you have nothing to share. Thanks for your time tho.

@Nusnus do you have any additional info you can share? Thanks in advance.